Mathlib/Analysis/SpecificLimits/Basic.lean

```diff
@@ -382,6 +382,44 @@ theorem ENNReal.tsum_geometric (r : ℝ≥0∞) : ∑' n : ℕ, r ^ n = (1 - r)
 theorem ENNReal.tsum_geometric_add_one (r : ℝ≥0∞) : ∑' n : ℕ, r ^ (n + 1) = r * (1 - r)⁻¹ := by
   simp only [_root_.pow_succ', ENNReal.tsum_mul_left, ENNReal.tsum_geometric]
 
+/-- Given x > 0, there is a sequence of positive reals summing to x. -/
```

Comment:
This lemma and the lemmas below feel a lot like `exists_pos_sum_of_countable` (in the same file!) and neighboring lemmas. Are you sure you need the new ones?

Reply:
Thanks for pointing out, I don't need the new ones. Sorry!